<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HPEHeadingDialogForm</class>
 <widget class="QWidget" name="HPEHeadingDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
	padding: 8px;
     color: gray;
     background-color: white;
     border-style: solid;
     border-width: 5px;
     border-radius: 8px;
     border-color: rgb(208, 208, 208);
     font: bold 15px;
}

QPushButton:hover
{
	/*border-color: gray;*/
     color: rgb(255, 255, 255);
     background-color: rgb(208, 208, 208);
}

QPushButton:checked
{
	color: rgb(255, 255, 255);
     background-color: gray;
}

QCheckBox {
	color: gray;
	font: bold 15px;
}

QCheckBox::indicator {
	width: 18px;
	height: 18px;
}

QCheckBox::indicator:unchecked {
	
	image: url(:/icons/resources/icons/ico_unchecked.png);
}


QCheckBox::indicator:checked {
	
	image: url(:/icons/resources/icons/ico_checked.png);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">border: 4px solid gray; border-radius: 8px;</string>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="buttonHeading1">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Heading 1</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+1</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="buttonHeading2">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Heading 2</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+2</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="buttonHeading3">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Heading 3</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+3</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="buttonHeading4">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Heading 4</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+4</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="buttonHeading5">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Heading 5</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+5</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="buttonHeading6">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Heading 6</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+6</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="headingEdit">
     <property name="styleSheet">
      <string notr="true">border: 4px solid gray;
border-radius: 8px;
padding: 0 4px;
selection-background-color: lightgray;
font-size: 20px;
font-weight: bold;
background-color: white;
color: gray;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>HEADING</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="addHorizontalRule">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Add Horizontal Rule</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
